herddb-core/src/test/java/herddb/core/RestartTest.java

```diff
@@ -713,4 +715,96 @@ public void run() {
 
     }
 
+    @Test
+    public void recoverTableAndIndexWithoutHash() throws Exception {
+
+        Path dataPath = folder.newFolder("data").toPath();
+        Path logsPath = folder.newFolder("logs").toPath();
+        Path metadataPath = folder.newFolder("metadata").toPath();
+        Path tmpDir = folder.newFolder("tmpDir").toPath();
+
+        String nodeId = "localhost";
+        Table table;
+        Index index;
+
+        boolean origHashChecksEnabled = ServerConfiguration.PROPERTY_HASH_CHECKS_ENABLED_DEFAULT;
+        boolean origHashWritesEnabled = ServerConfiguration.PROPERTY_HASH_WRITES_ENABLED_DEFAULT;
+        Whitebox.setInternalState(ServerConfiguration.class, "PROPERTY_HASH_CHECKS_ENABLED_DEFAULT", false);
```

Comment:
Why are you doing so?
There is no need now for powermock
You can simply call the FileDataStorageManager constructor with the wanted parameters.

I feel we can live without powermock, no need to add the dependency